src/NServiceBus.AcceptanceTests/Core/Recoverability/When_delayed_retries_with_serialization_exception.cs

```diff
@@ -18,17 +18,17 @@ public class When_delayed_retries_with_serialization_exception : NServiceBusAcce
                 .WithEndpoint<RetryEndpoint>(b => b
                     .When(session => session.SendLocal(new MessageToBeRetried()))
                     .DoNotFailOnErrorMessages())
-                .Done(c => c.DelayedRetryChecksum != default(byte))
+                .Done(c => c.FailedMessages.Any())
                 .Run();
 
-            Assert.AreEqual(context.OriginalBodyChecksum, context.DelayedRetryChecksum, "The body of the message sent to delayed retry should be the same as the original message coming off the queue");
+            var errorBody = context.FailedMessages.Single().Value.Single().Body;
+
+            CollectionAssert.AreEqual(context.OriginalBody, errorBody, "The body of the message sent to delayed retry should be the same as the original message coming off the queue");
```

Comment:
I think we don't need this variety of retry configuration. Let's go with the one which uses both retry options?